jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css

```diff
@@ -0,0 +1,163 @@
+.splitScreen {
+    height: calc(100vh);
```

Comment:
I believe you can just do 100vh here since you just have a constant value without mixing different measurements

Reply:
done